From 7b5146460873c22e47a6c45e330508f1b60532fd Mon Sep 17 00:00:00 2001
From: Michael Bestas <mikeioannina@cyanogenmod.org>
Date: Tue, 7 Jun 2016 00:06:53 +0300
Subject: [PATCH 42/67] Gallery2: Move & improve CAF strings

Change-Id: I3861ed5094307034fcec830fd1ff0595a75e84e1

Gallery2: Fix spelling error

Change-Id: I38c115a7a3a7148d5bcf5745261369cd57a431e9

Gallery: Fix translatability of CAF additions to filtershow

Change-Id: Ibcb2dfaf98ae367461365187c9549e544d119e90
---
 res/layout/filtershow_crop_button.xml         |   2 +-
 res/layout/filtershow_history_panel.xml       |   2 +-
 res/menu/filtershow_activity_menu.xml         |   2 +-
 res/menu/filtershow_menu_crop.xml             |   2 +-
 res/values/codeaurora_filtershow_strings.xml  | 150 ++++++++++++++++++
 res/values/codeaurora_strings.xml             | 113 +++++++++++--
 res/values/filtershow_strings.xml             | 124 +--------------
 res/values/strings.xml                        |  86 +---------
 res/xml/rtsp_settings_preferences.xml         |   4 +-
 .../filtershow/FilterShowActivity.java        |   6 +-
 10 files changed, 261 insertions(+), 230 deletions(-)
 create mode 100644 res/values/codeaurora_filtershow_strings.xml

diff --git a/res/layout/filtershow_crop_button.xml b/res/layout/filtershow_crop_button.xml
index b42d6b656..22c467ea6 100644
--- a/res/layout/filtershow_crop_button.xml
+++ b/res/layout/filtershow_crop_button.xml
@@ -24,4 +24,4 @@
     android:background="@drawable/filtershow_button_background"
     android:scaleType="centerInside"
     android:visibility="gone"
-    android:text="@string/aspectNone_effect" />
\ No newline at end of file
+    android:text="@string/aspectFree_effect" />
diff --git a/res/layout/filtershow_history_panel.xml b/res/layout/filtershow_history_panel.xml
index 392e39cbe..992d05a85 100644
--- a/res/layout/filtershow_history_panel.xml
+++ b/res/layout/filtershow_history_panel.xml
@@ -53,7 +53,7 @@
                 android:id="@+id/resetOperationsButton"
                 style="@style/FilterShowHistoryButton"
                 android:gravity="center"
-                android:text="@string/reset" />
+                android:text="@string/back_to_original" />
 
         <Button
                 android:id="@+id/saveOperationsButton"
diff --git a/res/menu/filtershow_activity_menu.xml b/res/menu/filtershow_activity_menu.xml
index d52c003a9..0f1c2066c 100644
--- a/res/menu/filtershow_activity_menu.xml
+++ b/res/menu/filtershow_activity_menu.xml
@@ -18,7 +18,7 @@
 <!--         android:title="@string/filtershow_redo"/> -->
     <item
         android:id="@+id/resetHistoryButton"
-        android:title="@string/reset"/>
+        android:title="@string/back_to_original"/>
 <!--     <item -->
 <!--         android:id="@+id/showImageStateButton" -->
 <!--         android:showAsAction="never" -->
diff --git a/res/menu/filtershow_menu_crop.xml b/res/menu/filtershow_menu_crop.xml
index f8ba3df42..94b06be5a 100644
--- a/res/menu/filtershow_menu_crop.xml
+++ b/res/menu/filtershow_menu_crop.xml
@@ -26,7 +26,7 @@
             android:title="@string/aspect9to16_effect"/>
         <item
             android:id="@+id/crop_menu_none"
-            android:title="@string/aspectNone_effect"/>
+            android:title="@string/aspectFree_effect"/>
         <item
             android:id="@+id/crop_menu_original"
             android:title="@string/aspectOriginal_effect"/>
diff --git a/res/values/codeaurora_filtershow_strings.xml b/res/values/codeaurora_filtershow_strings.xml
new file mode 100644
index 000000000..2244a874e
--- /dev/null
+++ b/res/values/codeaurora_filtershow_strings.xml
@@ -0,0 +1,150 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (C) 2012 The Android Open Source Project
+
+     Licensed under the Apache License, Version 2.0 (the "License");
+     you may not use this file except in compliance with the License.
+     You may obtain a copy of the License at
+
+          http://www.apache.org/licenses/LICENSE-2.0
+
+     Unless required by applicable law or agreed to in writing, software
+     distributed under the License is distributed on an "AS IS" BASIS,
+     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+     See the License for the specific language governing permissions and
+     limitations under the License.
+-->
+
+<resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
+    <!--  Text for filters that apply a frame to a picture [CHAR LIMIT=20] -->
+    <string name="frame">Frame <xliff:g id="number" example="1">%d</xliff:g></string>
+
+    <!--  String displayed when exiting with unsaved changes [CHAR LIMIT=NONE] -->
+    <string name="discard_message">Discard the changes you\'ve made to this image?</string>
+    <!--  String displayed when exiting with unsaved changes [CHAR LIMIT=NONE] -->
+    <string name="discard_title">Discard changes?</string>
+    <!--  String displayed when discard editor [CHAR LIMIT=NONE] -->
+    <string name="discard">Discard</string>
+
+    <!--  Text for the history panel reset button [CHAR LIMIT=20]-->
+    <string name="back_to_original">Back to original</string>
+
+    <!--  Label for the aspect None effect [CHAR LIMIT=15] -->
+    <string name="aspectFree_effect">Free</string>
+
+    <!--  Filters buttons -->
+
+    <!--  True Effect button -->
+    <string name="trueScanner" translatable="false">TrueScanner</string>
+    <!--  HazeBuster button -->
+    <string name="hazeBuster" translatable="false">HazeBuster</string>
+    <!--  SeeStraight button -->
+    <string name="seeStraight" translatable="false">SeeStraight</string>
+
+    <!--  Label for the "refocus" dual camera effect  [CHAR LIMIT=15] -->
+    <string name="focus">Focus</string>
+    <!--  Label for the "fusion" dual camera effect  [CHAR LIMIT=15] -->
+    <string name="fusion">Fusion</string>
+    <!--  Label for the "sketch" dual camera effect  [CHAR LIMIT=15] -->
+    <string name="sketch">Sketch</string>
+    <!--  Label for the "halo" dual camera effect  [CHAR LIMIT=15] -->
+    <string name="halo">Halo</string>
+    <!--  Label for the "zoom" dual camera effect  [CHAR LIMIT=15] -->
+    <string name="zoom">Zoom</string>
+    <!--  Label for the "motion" dual camera effect  [CHAR LIMIT=15] -->
+    <string name="motion">Motion</string>
+    <!--  Label for the "bw" dual camera effect  [CHAR LIMIT=15] -->
+    <string name="bw">B/W</string>
+    <!--  Label for the "blackboard" dual camera effect  [CHAR LIMIT=15] -->
+    <string name="blackboard">Blackboard</string>
+    <!--  Label for the "whiteboard" dual camera effect  [CHAR LIMIT=15] -->
+    <string name="whiteboard">Whiteboard</string>
+    <!--  Label for the "posterize" dual camera effect  [CHAR LIMIT=15] -->
+    <string name="posterize">Posterize</string>
+    <!--  Label for the "negative" dual camera effect  [CHAR LIMIT=15] -->
+    <string name="dc_negative">Negative</string>
+    <!--  Label for the "blur" true portrait effect  [CHAR LIMIT=15] -->
+    <string name="blur">Blur</string>
+    <!--  Label for the "motion blur" true portrait effect  [CHAR LIMIT=15] -->
+    <string name="motion_blur">Motion blur</string>
+
+    <string name="fusion_pick_point">Pick segment</string>
+    <string name="fusion_pick_underlay">Pick underlay</string>
+    <string name="dual_camera_effects">Dual camera effects</string>
+    <string name="dual_camera_effects_intro">Allows you to apply background effects, such as "Motion" and "Sketch". You can also merge two photos together for interesting effects. This feature works best when you have a short distance your camera and the main subject of you picture.</string>
+    <string name="dualcam_no_segment_toast">No segment found at this point</string>
+    <string name="dualcam_filter_not_supported">Dual camera filters not supported for this image</string>
+    <string name="trueportrait_intro">Allows you to enhance photos of people. You can apply effects to the background, such as "Blur" and "Sketch". You can also merge two photos together to create interesting backgrounds for your portrait photos.</string>
+    <string name="trueportrait_no_face">TruePortrait effects cannot be applied to this photo. Face not detected.</string>
+    <string name="trueportrait_edit_intro">With the touch-up tool, you can edit the foreground and the background. You can use your finger to paint in any areas of the foreground or background that were missed.</string>
+    <string name="trueportrait_edit_foreground_toast">Use your finger to paint in any areas of the foreground that were missed.</string>
+    <string name="trueportrait_edit_background_toast">Use your finger to paint in any areas of the background that were missed.</string>
+    <string name="trueportrait_edit_help">To zoom, use two fingers to pinch. You can also drag with two fingers to re-position the image. This can make it easier to fix any problematic areas.</string>
+    <string name="trueportrait_fusion_intro">Your foreground person can be copied and pasted onto any photo in your Gallery. You can try using different background photos to create new and interesting portraits.</string>
+    <string name="dualcam_fusion_intro">Your foreground can be copied and pasted onto any photo in your Gallery.\n\nYou can try using different background photos to create new and interesting portraits.</string>
+    <string name="do_not_show_again">Do not show again</string>
+    <string name="trueportrait_touch_up">Touch up</string>
+    <string name="fusion_pick_background">Select background photo</string>
+
+    <string name="color">Color</string>
+    <string name="ok_continue">Continue</string>
+    <string name="frames">Frames</string>
+    <string name="beautify">Beautify</string>
+    <string name="dualcam">Dual camera</string>
+    <string name="trueportrait" translatable="false">TruePortrait</string>
+    <string name="foreground">Foreground</string>
+    <string name="background">Background</string>
+    <string name="brush_size">Brush Size</string>
+
+    <string name="pref_dualcam_intro_show_key" translatable="false">pref_dualcam_intro_show_key</string>
+    <string name="pref_trueportrait_intro_show_key" translatable="false">pref_trueportrait_intro_show_key</string>
+    <string name="pref_trueportrait_edit_intro_show_key" translatable="false">pref_trueportrait_edit_intro_show_key</string>
+    <string name="pref_trueportrait_fusion_intro_show_key" translatable="false">pref_trueportrait_fusion_intro_show_key</string>
+    <string name="pref_trueportrait_fusion_underlay_key" translatable="false">pref_trueportrait_fusion_underlay_key</string>
+    <string name="pref_dualcam_fusion_underlay_key" translatable="false">pref_dualcam_fusion_underlay_key</string>
+    <string name="pref_dualcam_fusion_intro_show_key" translatable="false">pref_trueportrait_fusion_intro_show_key</string>
+
+    <!--Filtershow dialog -->
+    <string name="filtershow_dialog_title">Custom filter</string>
+    <string name="filtershow_dialog_content">Allows you to create a photo filter.\n\nSelect a photo you like and use it as a filter for other photos</string>
+    <string name="filtershow_dialog_original_photo">Original photo</string>
+    <string name="filtershow_dialog_apply_photo">Colors to apply</string>
+    <string name="filtershow_dialog_checkbox">Do not show again</string>
+    <string name="pref_filtergenerator_intro_show_key" translatable="false">pref_filtergenerator_intro_show_key</string>
+    <string name="filtershow_preset_title">Custom</string>
+    <string name="filtershow_preset_rename">Rename</string>
+    <string name="delete_before_exit">Delete filter?</string>
+    <string name="rename_before_exit">Rename filter</string>
+    <string name="filter_name">Filter name</string>
+    <string name="filter_name_notification">Name cannot be empty.</string>
+    <string name="filter_name_duplicate">Name already exists, please enter a different one.</string>
+    <string name="watermark_location">Location</string>
+    <string name="watermark_time">Time</string>
+    <string name="watermark_weather">Weather</string>
+    <string name="watermark_emotions">Emotions</string>
+    <string name="watermark_food">Food</string>
+    <string name="location_pin" translatable="false">LOCATION_PIN</string>
+    <string name="location_city" translatable="false">LOCATION_CITY</string>
+    <string name="location_hello" translatable="false">LOCATION_HELLO</string>
+    <string name="location_stamp" translatable="false">LOCATION_STAMP</string>
+    <string name="time_hourglass" translatable="false">TIME_HOURGLASS</string>
+    <string name="time_timestamp" translatable="false">TIME_TIMESTAMP</string>
+    <string name="time_sunrise" translatable="false">TIME_SUNRISE</string>
+    <string name="time_calendar" translatable="false">TIME_CALENDAR</string>
+    <string name="weather_rain" translatable="false">WEATHER_RAIN</string>
+    <string name="weather_snow" translatable="false">WEATHER_SNOW</string>
+    <string name="weather_sun" translatable="false">WEATHER_SUN</string>
+    <string name="weather_artistic_sun" translatable="false">WEATHER_ARTISTIC_SUN</string>
+    <string name="emotion_party" translatable="false">EMOTION_PARTY</string>
+    <string name="emotion_peace" translatable="false">EMOTION_PEACE</string>
+    <string name="emotion_cry" translatable="false">EMOTION_CRY</string>
+    <string name="emotion_happy" translatable="false">EMOTION_HAPPY</string>
+    <string name="food_fork_knife" translatable="false">FOOD_FORK_KNIFE</string>
+    <string name="food_tea_time" translatable="false">FOOD_TEA_TIME</string>
+    <string name="food_cheers" translatable="false">FOOD_CHEERS</string>
+    <string name="food_yum" translatable="false">FOOD_YUM</string>
+
+    <!--Filtershow Toast -->
+    <string name="no_faces_found">No faces found</string>
+    <string name="image_size_too_small">Image size too small</string>
+    <string name="not_support_gif">Gif files are not supported, please choose another photo</string>
+</resources>
diff --git a/res/values/codeaurora_strings.xml b/res/values/codeaurora_strings.xml
index 941365252..b05aaf45c 100644
--- a/res/values/codeaurora_strings.xml
+++ b/res/values/codeaurora_strings.xml
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!--
 Copyright (c) 2013-2014, The Linux Foundation. All rights reserved.
+Copyright (C) 2014-2016 The CyanogenMod Project
 
 Redistribution and use in source and binary forms, with or without
 modification, are permitted provided that the following conditions are
@@ -31,7 +32,7 @@ IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 <resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
     <!-- Strings for step option settins -->
     <string name="setp_option_name">Step</string>
-    <string name="step_option_desc">Customize fast forward/backward step</string>
+    <string name="step_option_desc">Customize fast forward/rewind step</string>
     <string name="setp_option_three_second">3 seconds</string>
     <string name="setp_option_six_second">6 seconds</string>
 
@@ -40,42 +41,120 @@ IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
     <string name="stereo">"Stereo"</string>
     <string name="speaker_on">Speaker ON</string>
     <string name="speaker_off">Speaker OFF</string>
-    <string name="speaker_need_headset">Please plug in headset first!</string>
+    <string name="speaker_need_headset">Plug in headphones to use this feature.</string>
     <string name="single_track">"Single track"</string>
-    <string name="media_detail">"Media details"</string>
-    <string name="input_url">"Input URL"</string>
-    <string name="streaming_settings">"Streaming Settings"</string>
+    <string name="input_url">"Enter URL"</string>
+    <string name="streaming_settings">"Streaming settings"</string>
     <string name="next">"Next"</string>
     <string name="previous">"Previous"</string>
 
-    <string name="rtp_rtcp">RTP/RTCP</string>
-    <string name="min_port">min port:</string>
-    <string name="max_port">max port:</string>
-    <string name="buffer_size">Buffer Size</string>
-    <string name="apn">Prefered APN</string>
+    <string name="rtp_rtcp" translatable="false">RTP/RTCP</string>
+    <string name="buffer_size">Buffer size</string>
+    <string name="apn">Preferred APN</string>
     <string name="rtp_min_port">Min port</string>
     <string name="rtp_max_port">Max port</string>
     <string name="set_rtp_min_port">Set min port</string>
     <string name="set_rtp_max_port">Set max port</string>
     <string name="set_buffer_size">Set buffer size</string>
-    <string name="set_apn">Select prefered apn</string>
+    <string name="set_apn">Select preferred APN</string>
     <string name="setting">Settings</string>
 
-    <string name="server_timeout_title">"Server Timeout"</string>
-    <string name="server_timeout_message">"Reconnect the server to play the video or not?"</string>
-    <string name="videoview_error_text_cannot_connect_retry">"Connection failed, try to reconnect %1$d..."</string>
+    <string name="server_timeout_title">"Server timeout"</string>
+    <string name="server_timeout_message">"Reconnect to the server to play the video?"</string>
+    <string name="videoview_error_text_cannot_connect_retry">"Connection failed, trying to reconnect to %1$d\u2026"</string>
     <string name="media_controller_live">Live</string>
     <string name="media_controller_playing">Playing</string>
-    <string name="media_controller_connecting">Connecting...</string>
+    <string name="media_controller_connecting">Connecting\u2026</string>
 
     <string name="bookmark_add">"Add bookmark"</string>
     <string name="bookmark_display">"Show bookmarks"</string>
     <string name="bookmark_empty">"No bookmarks"</string>
-    <string name="bookmark_exist">"Same bookmark in history"</string>
-    <string name="bookmark_add_success">"Add bookmark success"</string>
+    <string name="bookmark_exist">"Bookmark already exists"</string>
+    <string name="bookmark_add_success">"Bookmark added"</string>
     <string name="bookmark_list">"Bookmarks"</string>
     <string name="bookmark_title">"Title"</string>
     <string name="bookmark_location">"Location"</string>
     <string name="delete_all">"Delete all"</string>
     <string name="default_title">"Default title"</string>
+
+    <!-- A label describing to link website [CHAR LIMIT=30] -->
+    <string name="more_image">More images</string>
+    <string name="website_for_more_image" translatable="false"></string>
+
+    <!-- Toast if the muted video is not supported. [CHAR LIMIT=80] -->
+    <string name="mute_nosupport">Can not mute : video not supported</string>
+
+    <!-- The message is shown in toast when click showOnMap Menu and there is no map app -->
+    <string name="map_activity_not_found_err">There is no maps app installed to show the location.</string>
+
+    <plurals name="number_of_videos">
+        <item quantity="one">%1$d video</item>
+        <item quantity="other">%1$d videos</item>
+    </plurals>
+
+    <!-- The label for the bass boost knob of the audio effects dialog. -->
+    <string name="bass_boost_strength">Bass boost</string>
+
+    <!-- The label for the 3d effect knob of the audio effects dialog. -->
+    <string name="virtualizer_strength">3D effect</string>
+
+    <!-- The label for the audio effects menu. -->
+    <string name="audio_effects">Audio effects</string>
+
+    <!-- Toast if user attempts to control audio effects without headphones plugged in. -->
+    <string name="headset_plug">Plug in headphones for these effects.</string>
+
+    <!-- The title of the audio effects dialog. -->
+    <string name="audio_effects_dialog_title">Snapdragon Audio+</string>
+
+    <!-- Toast message for share items -->
+    <string name="cannot_share_items">Max selected items for share is 300</string>
+
+    <string name="share_dialogue_title">Share via</string>
+
+    <!-- When grouping photos by peoples, the label used for photos that don't
+         have people information in them [CHAR LIMIT=20]-->
+    <string name="no_faces">No faces</string>
+
+    <!-- The tips of trimming video -->
+    <string name="fail_trim">Sorry, this video file can not be trimmed</string>
+    <string name="can_not_trim">Only mp4 and 3gp files can be trimmed</string>
+
+    <string name="drm_license_info">DRM license info</string>
+
+    <string-array name="title_array_nav_items">
+        <item>Timeline</item>
+        <item>Albums</item>
+        <item>Videos</item>
+    </string-array>
+
+    <string name="navigation_drawer_title" translatable="false">@string/app_name</string>
+    <!-- Timeline screen title -->
+    <string name="timeline_title">Timeline</string>
+    <!-- Albums screen title -->
+    <string name="albums_title" translatable="false">@string/tab_albums</string>
+    <!-- Videos screen title -->
+    <string name="videos_title">Videos</string>
+    <string name="action_viewtype_list">List view</string>
+    <string name="action_viewtype_grid" translatable="false">@string/switch_photo_grid</string>
+    <string name="tvEmptyAlbum">No photos found</string>
+    <string name="tvEmptyVideos">No videos found</string>
+
+    <string name="text_makeup_whiten">Whiten</string>
+    <string name="text_makeup_Soften">Soften</string>
+    <string name="text_makeup_trimface">Trimface</string>
+    <string name="text_makeup_bigeye">Bigeye</string>
+    <string name="drawer_desc" translatable="false">drawer</string>
+    <string name="truescanner_normal">Normal</string>
+    <string name="truescanner_white">Whiteboard</string>
+    <string name="truescanner">TrueScanner</string>
+    <string name="truescanner_remove_glare">Remove Glare</string>
+    <string name="hazebuster_acts">Haze Buster</string>
+    <string name="seestraight_acts">See Straight</string>
+    <string name="seestraight_input_image_is_small">Input image is too small to process.</string>
+    <string name="seestraight_process_fail">Couldn\'t process the image.</string>
+
+    <string name="trim_video_exit_title">Discard changes?</string>
+    <string name="trim_video_exit_msg">Discard the changes made to this video?</string>
+    <string name="trim_video_exit_discard">DISCARD</string>
 </resources>
diff --git a/res/values/filtershow_strings.xml b/res/values/filtershow_strings.xml
index 18af9b452..98f32ebda 100644
--- a/res/values/filtershow_strings.xml
+++ b/res/values/filtershow_strings.xml
@@ -35,7 +35,7 @@
     <!--  Text to label an image as "original" [CHAR LIMIT=20] -->
     <string name="original">Original</string>
     <!--  Text for filters that apply a border to a picture [CHAR LIMIT=20] -->
-    <string name="borders" msgid="4461692156695893616">Frame</string>
+    <string name="borders" msgid="4461692156695893616">Borders</string>
     <!-- Text for the custom border filter [CHAR LIMIT=20] -->
     <string name="custom_border">Custom</string>
 
@@ -73,13 +73,11 @@
     <!--  Exit Dialog -->
 
     <!--  String displayed when exiting with unsaved changes [CHAR LIMIT=NONE] -->
-    <string name="unsaved">Discrad the changes you\'ve made to this image?</string>
+    <string name="unsaved">There are unsaved changes to this image.</string>
     <!--  String displayed when exiting with unsaved changes [CHAR LIMIT=NONE] -->
-    <string name="save_before_exit">Discard changes?</string>
+    <string name="save_before_exit">Do you want to save before exiting?</string>
     <!--  String displayed when saving and exiting editor [CHAR LIMIT=NONE] -->
     <string name="save_and_exit">Save and Exit</string>
-    <!--  String displayed when discard editor [CHAR LIMIT=NONE] -->
-    <string name="discard">Discard</string>
     <!--  String displayed when saving and exiting editor [CHAR LIMIT=NONE] -->
     <string name="save_and_processing">Processing full resolution image...</string>
     <!--  String displayed when exiting editor[CHAR LIMIT=NONE] -->
@@ -90,7 +88,7 @@
     <!--  Text for the history panel title [CHAR LIMIT=50] -->
     <string name="history">History</string>
     <!--  Text for the history panel reset button [CHAR LIMIT=20]-->
-    <string name="reset">Back to original</string>
+    <string name="reset">Reset</string>
     <!--  Text for the original image[CHAR LIMIT=20]-->
     <string name="history_original">@string/original</string>
 
@@ -124,7 +122,7 @@
     <!--  Label for the aspect 1:1 effect [CHAR LIMIT=15] -->
     <string name="aspect9to16_effect">16:9</string>
     <!--  Label for the aspect None effect [CHAR LIMIT=15] -->
-    <string name="aspectNone_effect">Free</string>
+    <string name="aspectNone_effect">None</string>
     <!--  Label for the aspect None effect [CHAR LIMIT=15] -->
     <string name="aspectOriginal_effect">@string/original</string>
     <!-- Label for when the aspect ratio is fixed to a value [CHAR LIMIT=15] -->
@@ -134,12 +132,6 @@
 
     <!--  Filters buttons -->
 
-    <!--  True Effect button -->
-    <string name="trueScanner">True Scanner</string>
-    <!--  HazeBuster button -->
-    <string name="hazeBuster">Haze Buster</string>
-    <!--  SeeStraight button -->
-    <string name="seeStraight">See Straight</string>
     <!--  Label for the image exposure (brightness) filter button [CHAR LIMIT=10] -->
     <string name="exposure" msgid="1229093066434614811">Exposure</string>
     <!--  Label for the image sharpness (clarity, distinctness) filter button [CHAR LIMIT=10] -->
@@ -200,32 +192,6 @@
     <string name="downsample">Downsample</string>
     <!--  Label for the "neutral density graduated filter" filter effect  [CHAR LIMIT=15] -->
     <string name="grad">Graduated</string>
-    <!--  Label for the "refocus" dual camera effect  [CHAR LIMIT=15] -->
-    <string name="focus">Focus</string>
-    <!--  Label for the "fusion" dual camera effect  [CHAR LIMIT=15] -->
-    <string name="fusion">Fusion</string>
-    <!--  Label for the "sketch" dual camera effect  [CHAR LIMIT=15] -->
-    <string name="sketch">Sketch</string>
-    <!--  Label for the "halo" dual camera effect  [CHAR LIMIT=15] -->
-    <string name="halo">Halo</string>
-    <!--  Label for the "zoom" dual camera effect  [CHAR LIMIT=15] -->
-    <string name="zoom">Zoom</string>
-    <!--  Label for the "motion" dual camera effect  [CHAR LIMIT=15] -->
-    <string name="motion">Motion</string>
-    <!--  Label for the "bw" dual camera effect  [CHAR LIMIT=15] -->
-    <string name="bw">B/W</string>
-    <!--  Label for the "blackboard" dual camera effect  [CHAR LIMIT=15] -->
-    <string name="blackboard">Blackboard</string>
-    <!--  Label for the "whiteboard" dual camera effect  [CHAR LIMIT=15] -->
-    <string name="whiteboard">Whiteboard</string>
-    <!--  Label for the "posterize" dual camera effect  [CHAR LIMIT=15] -->
-    <string name="posterize">Posterize</string>
-    <!--  Label for the "negative" dual camera effect  [CHAR LIMIT=15] -->
-    <string name="dc_negative">Negative</string>
-    <!--  Label for the "blur" true portrait effect  [CHAR LIMIT=15] -->
-    <string name="blur">Blur</string>
-    <!--  Label for the "motion blur" true portrait effect  [CHAR LIMIT=15] -->
-    <string name="motion_blur">Motion Blur</string>
     <!--  Label for the Brightness effect  [CHAR LIMIT=20] -->
     <string name="editor_grad_brightness">Brightness</string>
     <!--  Label for the Contrast filter effect  [CHAR LIMIT=20] -->
@@ -362,84 +328,4 @@
     <!-- Label for exif tags in the information panel [CHAR LIMIT=50] -->
     <string name="filtershow_exif_copyright">Copyright</string>
 
-    <string name="fusion_pick_point">Pick Segment</string>
-    <string name="fusion_pick_underlay">Pick Underlay</string>
-    <string name="dual_camera_effects">Dual Camera Effects</string>
-    <string name="dual_camera_effects_intro">Allows you to apply background effects, such as "Motion" and "Sketch". You can also merge two photos together for interesting effects. This feature works best when you have a short distance your camera and the main subject of you picture.</string>
-    <string name="dualcam_no_segment_toast">No segment found at this point</string>
-    <string name="dualcam_filter_not_supported">Dual camera filters not supported for this image</string>
-    <string name="trueportrait_intro">Allows you to enhance photos of people. You can apply effects to the background, such as \"blur\" and \"sketch\". You can also merge two photos together to create interesting backgrounds for your portrait photos.</string>
-    <string name="trueportrait_no_face">True Portrait effects cannot be applied to this photo. Face not detected.</string>
-    <string name="trueportrait_edit_intro">With the Touch-Up tool, you can edit the foreground and the background. You can user your finger to paint in any areas of the foreground or background that were missed.</string>
-    <string name="trueportrait_edit_foreground_toast">Use your finger to paint in any areas of the foreground that were missed.</string>
-    <string name="trueportrait_edit_background_toast">Use your finger to paint in any areas of the background that were missed.</string>
-    <string name="trueportrait_edit_help">To zoom, use two fingers to pinch open or closed. You can also drag with two fingers to re-position the image. This can make it easier to fix any problem areas.</string>
-    <string name="trueportrait_fusion_intro">Your foreground person can be copied and pasted onto any photo in your Gallery. You can try using different background photos to create new and interesting portraits.</string>
-    <string name="dualcam_fusion_intro">Your foreground can be copied and pasted onto any photo in your Gallery. \n\nYou can try using different background photos to create new and interesting portraits.</string>
-    <string name="do_not_show_again">Do not show again</string>
-    <string name="trueportrait_touch_up">Touch Up</string>
-    <string name="fusion_pick_background">Select Background Photo</string>
-
-    <string name="color">Color</string>
-    <string name="ok_continue">Continue</string>
-    <string name="frames">Frames</string>
-    <string name="beautify">Beautify</string>
-    <string name="dualcam">Dual Camera</string>
-    <string name="trueportrait">True Portrait</string>
-    <string name="foreground">Foreground</string>
-    <string name="background">Background</string>
-    <string name="brush_size">Brush Size</string>
-
-    <string name="pref_dualcam_intro_show_key" translatable="false">pref_dualcam_intro_show_key</string>
-    <string name="pref_trueportrait_intro_show_key" translatable="false">pref_trueportrait_intro_show_key</string>
-    <string name="pref_trueportrait_edit_intro_show_key" translatable="false">pref_trueportrait_edit_intro_show_key</string>
-    <string name="pref_trueportrait_fusion_intro_show_key" translatable="false">pref_trueportrait_fusion_intro_show_key</string>
-    <string name="pref_trueportrait_fusion_underlay_key" translatable="false">pref_trueportrait_fusion_underlay_key</string>
-    <string name="pref_dualcam_fusion_underlay_key" translatable="false">pref_dualcam_fusion_underlay_key</string>
-    <string name="pref_dualcam_fusion_intro_show_key" translatable="false">pref_trueportrait_fusion_intro_show_key</string>
-
-    <!--Filtershow dialog -->
-    <string name="filtershow_dialog_title">Custom Filter</string>
-    <string name="filtershow_dialog_content">Allows you to create a photo filter.\n\nSelect a photo you like and use it as a filter for other photos</string>
-    <string name="filtershow_dialog_original_photo">Original Photo</string>
-    <string name="filtershow_dialog_apply_photo">Colors to Apply</string>
-    <string name="filtershow_dialog_checkbox">Do not show again</string>
-    <string name="pref_filtergenerator_intro_show_key" translatable="false">pref_filtergenerator_intro_show_key</string>
-    <string name="filtershow_preset_title">Custom</string>
-    <string name="filtershow_preset_rename">Rename</string>
-    <string name="delete_before_exit">Delete Filter?</string>
-    <string name="rename_before_exit">Rename Filter</string>
-    <string name="filter_name">Filter name</string>
-    <string name="filter_name_notification">Name cannot be NULL.</string>
-    <string name="filter_name_duplicate">Name already existed, please input another one.</string>
-    <string name="watermark_location">Location</string>
-    <string name="watermark_time">Time</string>
-    <string name="watermark_weather">Weather</string>
-    <string name="watermark_emotions">Emotions</string>
-    <string name="watermark_food">Food</string>
-    <string name="location_pin">LOCATION_PIN</string>
-    <string name="location_city">LOCATION_CITY</string>
-    <string name="location_hello">LOCATION_HELLO</string>
-    <string name="location_stamp">LOCATION_STAMP</string>
-    <string name="time_hourglass">TIME_HOURGLASS</string>
-    <string name="time_timestamp">TIME_TIMESTAMP</string>
-    <string name="time_sunrise">TIME_SUNRISE</string>
-    <string name="time_calendar">TIME_CALENDAR</string>
-    <string name="weather_rain">WEATHER_RAIN</string>
-    <string name="weather_snow">WEATHER_SNOW</string>
-    <string name="weather_sun">WEATHER_SUN</string>
-    <string name="weather_artistic_sun">WEATHER_ARTISTIC_SUN</string>
-    <string name="emotion_party">EMOTION_PARTY</string>
-    <string name="emotion_peace">EMOTION_PEACE</string>
-    <string name="emotion_cry">EMOTION_CRY</string>
-    <string name="emotion_happy">EMOTION_HAPPY</string>
-    <string name="food_fork_knife">FOOD_FORK_KNIFE</string>
-    <string name="food_tea_time">FOOD_TEA_TIME</string>
-    <string name="food_cheers">FOOD_CHEERS</string>
-    <string name="food_yum">FOOD_YUM</string>
-
-    <!--Filtershow Toast -->
-    <string name="no_faces_found">No Faces Found</string>
-    <string name="image_size_too_small">Image size too small</string>
-    <string name="not_support_gif">do not support gif extension, please pick again</string>
 </resources>
diff --git a/res/values/strings.xml b/res/values/strings.xml
index c4b53fe32..570014e3e 100755
--- a/res/values/strings.xml
+++ b/res/values/strings.xml
@@ -1,8 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!-- Copyright (c) 2014, 2016, The Linux Foundation. All rights reserved.
-     Not a Contribution.
-
-     Copyright (C) 2007 The Android Open Source Project
+<!-- Copyright (C) 2007 The Android Open Source Project
 
      Licensed under the Apache License, Version 2.0 (the "License");
      you may not use this file except in compliance with the License.
@@ -106,8 +103,6 @@
     <!-- Toast/alert after saving wallpaper -->
     <string name="wallpaper">Setting wallpaper\u2026</string>
     <string name="camera_setas_wallpaper">Wallpaper</string>
-    <!-- Toast message for share items -->
-    <string name="cannot_share_items">Max selected items for share is 300</string>
 
     <!-- Details dialog "OK" button. Dismisses dialog. -->
     <string name="delete">Delete</string>
@@ -119,7 +114,6 @@
     <string name="confirm">Confirm</string>
     <string name="cancel">Cancel</string>
     <string name="share">Share</string>
-    <string name="share_dialogue_title">Share via</string>
     <string name="share_panorama">Share panorama</string>
     <string name="share_as_photo">Share as photo</string>
 
@@ -239,10 +233,6 @@
          have location information in them [CHAR LIMIT=20]-->
     <string name="no_location">No location</string>
 
-    <!-- When grouping photos by peoples, the label used for photos that don't
-        have people information in them [CHAR LIMIT=20]-->
-    <string name="no_faces">No faces</string>
-
     <!-- This toast message is shown when network connection is lost while doing clustering -->
     <string name="no_connectivity">Some locations couldn\'t be identified due to network problems.</string>
 
@@ -288,10 +278,6 @@
          some albums to be viewable offline [CHAR LIMIT=30] -->
     <string name="make_available_offline">Make available offline</string>
 
-    <!-- A label describing to link website [CHAR LIMIT=30] -->
-    <string name="more_image">More Images</string>
-    <string name="website_for_more_image"></string>
-
     <!-- A label of a menu item for user to sync the content [CHAR LIMIT=30] -->
     <string name="sync_picasa_albums">Refresh</string>
 
@@ -536,9 +522,6 @@
     <!-- Toast if the trimmed video is too short to trim. [CHAR LIMIT=80] -->
     <string name="trim_too_short">Can not trim : target video is too short</string>
 
-    <!-- Toast if the muted video is not supported. [CHAR LIMIT=80] -->
-    <string name="mute_nosupport">Can not mute : not supported video</string>
-
     <!-- Text to show with progress bar while stitching in Gallery -->
     <string name="pano_progress_text">Rendering panorama</string>
 
@@ -1111,9 +1094,6 @@ CHAR LIMIT = NONE] -->
     <!-- Positive answer for first run dialog asking if the user wants to remember photo locations [CHAR LIMIT = 20] -->
     <string name="remember_location_yes">Yes</string>
 
-    <!-- The message is shown in toast when click showOnMap Menu and there is no map app -->
-    <string name="map_activity_not_found_err">There is no map app for show location.</string>
-
     <!-- Menu item to launch the camera app [CHAR LIMIT=25] -->
     <string name="menu_camera">Camera</string>
     <!-- Menu item to search for photos [CHAR LIMIT=25] -->
@@ -1135,68 +1115,4 @@ CHAR LIMIT = NONE] -->
         <item quantity="one">%1$d photo</item>
         <item quantity="other">%1$d photos</item>
     </plurals>
-
-    <plurals name="number_of_videos">
-        <item quantity="one">%1$d video</item>
-        <item quantity="other">%1$d videos</item>
-    </plurals>
-
-    <!-- The label for the bass boost knob of the audio effects dialog. -->
-    <string name="bass_boost_strength">Bass boost</string>
-
-    <!-- The label for the 3d effect knob of the audio effects dialog. -->
-    <string name="virtualizer_strength">Surround sound</string>
-
-    <!-- The label for the audio effects menu. -->
-    <string name="audio_effects">Audio effects</string>
-
-    <!-- Toast if user attempts to control audio effects without headphones plugged in. -->
-    <string name="headset_plug">Plug in headphones for these effects.</string>
-
-    <!-- The title of the audio effects dialog. -->
-    <string name="audio_effects_dialog_title">Snapdragon Audio+</string>
-
-    <!-- The tips of trimming video -->
-    <string name="fail_trim">Sorry, this video file can not be trimmed</string>
-    <string name="can_not_trim">Only mp4 and 3gp file can be trimmed</string>
-
-    <string name="no_permission_for_drm">"Sorry, Drm file can not be shared"</string>
-    <string name="drm_license_info">Drm license info</string>
-    <string name="mimetype_not_supported">This file can not be played</string>
-    <string name="action_consumes_rights">Rights will be consumed for playing this media</string>
-    <string name="text_makeup_whiten">Whiten</string>
-    <string name="text_makeup_Soften">Soften</string>
-
-    <string-array name="title_array_nav_items">
-        <item>@string/timeline_title</item>
-        <item>@string/albums_title</item>
-        <item>@string/videos_title</item>
-    </string-array>
-
-    <string name="navigation_drawer_title" translatable="false">@string/app_name</string>
-    <!-- Timeline screen title -->
-    <string name="timeline_title">Timeline</string>
-    <!-- Albums screen title -->
-    <string name="albums_title" translatable="false">@string/tab_albums</string>
-    <!-- Videos screen title -->
-    <string name="videos_title">Videos</string>
-    <string name="action_viewtype_list">List view</string>
-    <string name="action_viewtype_grid" translatable="false">@string/switch_photo_grid</string>
-    <string name="tvEmptyAlbum">No photos found</string>
-    <string name="tvEmptyVideos">No videos found</string>
-    <string name="text_makeup_trimface">Trimface</string>
-    <string name="text_makeup_bigeye">Bigeye</string>
-    <string name="drawer_desc" translatable="false">drawer</string>
-    <string name="truescanner_normal">Normal</string>
-    <string name="truescanner_white">WhiteBoard</string>
-    <string name="truescanner">TrueScanner</string>
-    <string name="truescanner_remove_glare">Remove Glare</string>
-    <string name="hazebuster_acts">Haze Buster</string>
-    <string name="seestraight_acts">See Straight</string>
-    <string name="seestraight_input_image_is_small">Input image is too small to process.</string>
-    <string name="seestraight_process_fail">Couldn\'t process the image.</string>
-
-    <string name="trim_video_exit_title">Discard changes?</string>
-    <string name="trim_video_exit_msg">Discard the changes made to this video?</string>
-    <string name="trim_video_exit_discard">DISCARD</string>
 </resources>
diff --git a/res/xml/rtsp_settings_preferences.xml b/res/xml/rtsp_settings_preferences.xml
index 13842d34e..895fb7d55 100755
--- a/res/xml/rtsp_settings_preferences.xml
+++ b/res/xml/rtsp_settings_preferences.xml
@@ -11,7 +11,7 @@
             android:summary=""
             android:title="@string/rtp_min_port" />
         <EditTextPreference
-            android:dialogTitle="@string/rtp_max_port"
+            android:dialogTitle="@string/set_rtp_max_port"
             android:inputType="number"
             android:key="rtp_max_port"
             android:maxLength="5"
@@ -33,7 +33,7 @@
         android:key="apn_category">
         <PreferenceScreen
             android:key="apn"
-            android:title="@string/apn" >
+            android:title="@string/set_apn" >
         </PreferenceScreen>
     </PreferenceCategory>
 
diff --git a/src/com/android/gallery3d/filtershow/FilterShowActivity.java b/src/com/android/gallery3d/filtershow/FilterShowActivity.java
index 7e0f3ca23..466e19ba3 100755
--- a/src/com/android/gallery3d/filtershow/FilterShowActivity.java
+++ b/src/com/android/gallery3d/filtershow/FilterShowActivity.java
@@ -1281,7 +1281,7 @@ DialogInterface.OnDismissListener, PopupMenu.OnDismissListener{
 
         for (int i = 0; i < borders.size(); i++) {
             FilterRepresentation filter = borders.get(i);
-            filter.setName(getString(R.string.borders) + "" + i);
+            filter.setName(getString(R.string.frame, i));
             if (i == 0) {
                 filter.setName(getString(R.string.none));
             }
@@ -2514,8 +2514,8 @@ DialogInterface.OnDismissListener, PopupMenu.OnDismissListener{
 
     private void createBackDialog() {
         mBackAlertDialogBuilder = new AlertDialog.Builder(this);
-        mBackAlertDialogBuilder.setMessage(R.string.unsaved).setTitle(
-                R.string.save_before_exit);
+        mBackAlertDialogBuilder.setMessage(R.string.discard_message).setTitle(
+                R.string.discard_title);
         mBackAlertDialogBuilder.setPositiveButton(mPopUpText,
                 new DialogInterface.OnClickListener() {
             @Override
-- 
2.17.1

